### PR TITLE
Update `HumanitZ` egg

### PR DESCRIPTION
**Pull request by vp-en**
# Description

Changes:
- Added a missing variable, `LOOT_RESPAWN`. Apparently missed that in the original commit.
- Added a description for the variable `BUILDING_HEALTH`, now that it has been added to the official docs
- Added two new variables, `ALLOW_DISMANTLE` and `ALLOW_HOUSE_DISMANTLE`

## Checklist for all submissions

* [X] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [X] Have you tested and reviewed your changes with confidence that everything works?
* [X] Did you branch your changes and PR from that branch and not from your master branch?

* [X] You verify that the start command applied does not use a shell script
* [X] The egg was exported from the panel

### CODE REVIEW

File: game_eggs/steamcmd_servers/humanitz/egg-humanit-z.json [hunk 1]
```diff
@@ -230,6 +230,16 @@
         },
         {
             "name": "Loot Respawn",
+            "description": "1=Loot does respawn, 0=Never respawn",
+            "env_variable": "LOOT_RESPAWN",
+            "default_value": "1",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|numeric",
```
Comment (QuintenQVD0) [inline game_eggs/steamcmd_servers/humanitz/egg-humanit-z.json:238]: Set this to boolean validation.

File: game_eggs/steamcmd_servers/humanitz/egg-humanit-z.json [hunk 2]
```diff
@@ -467,6 +477,26 @@
...
+        },
+        {
+            "name": "Allow Dismantle",
+            "description": "1 = Enable players to dismantle their own buildings, 0 = Disable",
+            "env_variable": "ALLOW_DISMANTLE",
+            "default_value": "1",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|numeric",
+            "field_type": "text"
```
Comment (QuintenQVD0) [inline game_eggs/steamcmd_servers/humanitz/egg-humanit-z.json:489]: Same here

File: game_eggs/steamcmd_servers/humanitz/egg-humanit-z.json [hunk 3]
```diff
@@ -467,6 +477,26 @@
...
+        },
+        {
+            "name": "Allow House Dismantle",
+            "description": "1 = Players are able to dismantle house props, 0 = Disable",
+            "env_variable": "ALLOW_HOUSE_DISMANTLE",
+            "default_value": "1",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|numeric",
```
Comment (QuintenQVD0) [inline game_eggs/steamcmd_servers/humanitz/egg-humanit-z.json:498]: Same

**Review by QuintenQVD0 — CHANGES_REQUESTED** [review]

File: game_eggs/steamcmd_servers/humanitz/egg-humanit-z.json (hunk 3 above)
Comment (vp-en) [reply game_eggs/steamcmd_servers/humanitz/egg-humanit-z.json:498]: Oh yeah, whoops. I'll fix all of them.

**Review by QuintenQVD0 — APPROVED** [review]
LGTM! Thanks

**Pull request closed without merging**